### ISSUE TITLE
Edit Links cannot display all possible links in some cases

### ISSUE THREAD

**Issue by rgschmitz1** [BioDepot/BioDepot-workflow-builder#38]
When multiple links are available in the "Edit Links" window there is no option to scroll through the available links, the only option is to make the Bwb window workspace larger by zooming out and maximizing the Edit Links window.  There needs to be either a scroll bar added to accommodate multiple links or some mechanism to select links from another mechanism (drop down menu?)
![image](https://user-images.githubusercontent.com/14095796/119214631-2e930f00-ba7d-11eb-8f47-6ef25eb74c95.png)

**Comment by lhhunghimself** [comment]
Yup. the interface is inherited from OrangeML and is designed for far fewer inputs/outputs. I think that a subsetting mechanism is probably easier to implement than changing the graphics. I will look into this when I get some excess bandwidth.
